### PR TITLE
Reduction clause onto the simd line.

### PR DESCRIPTION
**Pull request by pvelesko**
reduction clause needs to go onto each #pragma simd aligned  line otherwise vectorization is broken

### CODE REVIEW

**Comment by FabioLuporini** [comment]
I think I'm missing something... why just adding a TODO?

**Comment by pvelesko** [comment]
Mathias asked to make a PR. I don't have time to fix this

**Comment by pvelesko** [comment]
@pp1336  can you change this back to float64 so it would pass travis and merge?

**Comment by pp1336** [comment]
sure

**Comment by FabioLuporini** [comment]
what's stopping this from merge?

**Comment by vincepandolfo** [comment]
I've cleaned up the git history. Should be good to go when travis is green

(removed the TODO commit, and the two precision ones)

File: examples/acoustic_example.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def source(t, f0):
 receiver_coords[:, 2] = location[2]
 data.set_receiver_pos(receiver_coords)
 data.set_shape(nt, 101)
-Acoustic = Acoustic_cg(model, data)
+Acoustic = Acoustic_cg(model, data, auto_tune=True)
```
Comment (mlange05) [inline examples/acoustic_example.py]: Why are we now defaulting to `auto_tune=True`? Isn't that still computationally expensive? Also, how would a user over-write that?

Comment (vincepandolfo) [inline examples/acoustic_example.py]: Can this be removed from this PR and discussed somewhere else? Shouldn't be related anyway

**Pull request merged**